src/ai/backend/web/server.py

```diff
@@ -94,6 +109,9 @@ async def static_handler(request: web.Request) -> web.StreamResponse:
             }),
             content_type="application/problem+json",
         )
+    if file_path.suffix in template_extension_list:
```

Comment:
I have checked the following two cases:

1. When a request is made without an extension, such as `http://127.0.0.1:8090/`, the response should be HTML. However, it seems that the nonce injection logic is not being applied in this case.
2. When accessing `http://127.0.0.1:8090/index.html`,  extension check condition is not passing. If I remove the condition, it works properly.